ext/scripts/install_scripts/install_via_r_remotes.pl

```diff
@@ -56,9 +56,9 @@ =head1 SYNOPSIS
 
 my $combining_template = "library(remotes)\n<<<<0>>>>";
 my @separators = ("\n");
-my @templates = ('install_version("<<<<0>>>>",NULL,repos="http://cran.r-project.org")');
+my @templates = ('install_version("<<<<0>>>>",NULL,repos="http://cran.r-project.org", Ncpus=4)');
```

Comment:
Maybe create a ticket to replace with:
```suggestion
my @templates = ('install_version("<<<<0>>>>",NULL,repos="http://cran.r-project.org", Ncpus=parallel::detectCores())');

```
later.

Reply:
Created https://github.com/exasol/script-languages-release/issues/468